product_code/hyperledger/api/src/test/scala/ChaincodeQuickAccessTests.scala

```diff
@@ -1,17 +1,20 @@
 import de.upb.cs.uc4.hyperledger.ChaincodeQuickAccess
-import org.scalatest.FunSuite
+import org.scalatest.wordspec.AnyWordSpec
 
-class ChaincodeQuickAccessTests extends FunSuite {
+class ChaincodeQuickAccessTests extends AnyWordSpec {
 
-  test("Check listing all courses") {
-    // retrieve quickAccess result
-    val result = ChaincodeQuickAccess.getAllCourses();
+  "The ChainCodeQuickAccess" when {
+    "accessed" should {
+      "Allow for listing all courses" in {
+        // retrieve quickAccess result
+        val result = ChaincodeQuickAccess.getAllCourses();
 
-    // debug help
-    println(result)
+        // debug help
+        println(result)
 
-    // perform tests
-    assert(result != null, "result was null")
+        // perform tests
+        assert(result != null, "result was null")

```

Comment:
We agreed to use Matchers with WordSpec for Testing

Reply:
fixed in [https://github.com/upb-uc4/University-Credits-4.0/pull/121/commits/710923f1f36afa50fc8faff88e18af10d9ceb896](https://github.com/upb-uc4/University-Credits-4.0/pull/121/commits/710923f1f36afa50fc8faff88e18af10d9ceb896)